imports/plugins/core/product/server/no-meteor/schemas/schema.graphql

```diff
@@ -6,3 +6,68 @@ type ProductConfiguration {
   "The ProductVariant ID"
   productVariantId: ID!
 }
+
+"Operation"
+type Operation {
+  "Id of document"
+  _id: String
+}
+
+"A bulk write error type"
+type WriteError {
+  "Error code"
+  code: Int
+
+  "Error message"
+  errmsg: String
+
+  "Index of of error"
+  index: Int
+
+  "Failed operation info"
+  op: Operation
+}
+
+"Input for adding tags to products in bulk"
+input ProductsBulkOperationInput {
+  "An optional string identifying the mutation call, which will be returned in the response payload"
+  clientMutationId: String
+
+  "An array of product ids to which an array of tags will be added"
+  productIds: [ID]
+
+  "An array of tag ids to add to an array of products"
```

Comment:
ids -> IDs, same on the description above

Reply:
updated